lib/apis/fetch.js

```diff
@@ -11,7 +11,10 @@ export default (url, options = {}) => {
 
     request(url, opts, (err, response) => {
       if (!!err || response.statusCode !== 200) {
-        return reject(err || new Error(response.body));
+        if (err) return reject(err);
+        const error = new Error(`${response.request.uri.href}: ${response.body}`);
```

Comment:
is it safe to assume that all these properties (`response.request.uri.href`) will always exist?